tests/api.c

```diff
@@ -122,6 +122,22 @@
     #include <wolfssl/wolfcrypt/hc128.h>
 #endif
 
+#ifndef NO_DSA
+    #include <wolfssl/wolfcrypt/dsa.h>
+    #ifndef BYTE1024
+        #define BYTE1024 1024
```

Comment:
For consistency in naming, I may call these ONEK_BUF and TWOK_BUF (to match the FOURK_BUF name).